portfolio/src/main/webapp/style.css

```diff
@@ -51,72 +51,72 @@ html {
 .headline-text {
     border: 5px solid white;
     color: white;
-    font: 2em/1.2 Arial, Helvetica, sans-serif;
+    font: 2em/1.2 Roboto;
     font-weight: bolder;
-    padding:40px;
+    padding: 40px;
     position: relative;
     text-align: center;
     z-index: 20;
 }
 
-.about {
-    background-color:white;
+.profile{
```

Comment:
Talked with Luke and we decided that we are going to be a bit more nitpicky leading up to working together. That way we don't have to worry about that when working on code people are editing together.

In that vein, can you fix the inconsistency in spacing before the { in this file? Some styles have a space, some don't. I'd prefer a space everywhere (like .headline-text for example is good but .profile is not).

Also make sure everything's alphabetical.

The reason for this nits is that it can significantly reduce merge conflicts later.